x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -53,6 +55,11 @@ export const HostIsolationExceptionsList = () => {
 
   const { isLoading, data, error, refetch } = useFetchHostIsolationExceptionsList();
 
+  // load the list of policies>
+  const policiesRequest = useGetEndpointSpecificPolicies({
+    onError: () => {},
```

Comment:
why is this an empty error?

Reply:
good catch! l forgot to add a toast here.